### ISSUE TITLE
[Discussion] Launching multiple SGLang Diffusion Replica on the same GPU

### ISSUE THREAD

**Issue by zhaochenyang20** [zhaochenyang20/sglang-diffusion-routing#12]
I found that running a single SGLang Diffusion server with Qwen Image takes roughly 20GB at the beginning, then gradually goes to 56GB on H200.

1. Why would the GPU consumption go up? If this is expected, it would be dangerous. 🤔 Does it actually have an upper bound, or do we have similar control logits like LLM with parameters like `mem_static`?
2. Given the pick VRAM of a single SGLang diffusion server of 56GB on one H200, does it make sense to place two SGLang diffusion servers on the same GPU?
- Is this meaningful? In my understanding, diffusion models are like prefill workload, compute-bound. Putting two servers on one GPU does not help since the compute is bounded.
- Is it safe? As I asked in 1, if the memory continuously grows up, this will be pretty dangerous.
